lib/cookie.js

```diff
@@ -725,6 +753,7 @@ Cookie.prototype.domain = null;
 Cookie.prototype.path = null;
 Cookie.prototype.secure = false;
 Cookie.prototype.httpOnly = false;
+Cookie.prototype.sameSite = "none";
```

Comment:
I should stick a comment on this about this step in Section 5.4:

> 13.  If the cookie-attribute-list contains an attribute with an
>        attribute-name of "SameSite", set the cookie's same-site-flag to
>        attribute-value (i.e. either "Strict" or "Lax").  Otherwise, set
>        the cookie's same-site-flag to "None".

Reply:
Nevermind, have that in the code below already.

Reply:
I see that. :) No worries.